thorlcr/activities/choosesets/thchoosesetsslave.cpp

```diff
@@ -250,6 +250,7 @@ class InputCounter : public CSimpleInterface, implements IThorDataLink
     virtual CActivityBase *queryFromActivity();
     virtual void dataLinkSerialize(MemoryBuffer &mb);
     unsigned __int64 queryTotalCycles() const;
+    virtual void debugRequest(MemoryBuffer &msg);
```

Comment:
minor: would be good to make the other IThorDataLink interface definitions here, declared with virtual as well at this juncture.